app/services/api/v3/participant_declarations_query.rb

```diff
@@ -57,6 +57,13 @@ def participant_declarations_from(paginated_join)
         scope
       end
 
+      def find(id)
```

Comment:
suggestion: could we name it maybe `participant_declaration` similar to other services singular methods?

Reply:
Updated